cmd/rook/ceph/operator.go

```diff
@@ -28,9 +29,22 @@ import (
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	"github.com/rook/rook/pkg/util/flags"
 	"github.com/spf13/cobra"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-const containerName = "rook-ceph-operator"
+const (
+	containerName     = "rook-ceph-operator"
+	namespace         = "rook-ceph"
```

Comment:
This doesn't feel right to assume this. The namespace should be discovered.

Reply:
You can get the namespace of the operator from this:
```
os.Getenv(k8sutil.PodNamespaceEnvVar)
```

Reply:
Fixed. Thanks @travisn